tests/lib/cli.js

```diff
@@ -171,12 +219,16 @@ describe("cli", function() {
 
     describe("when calling execute more than once", function() {
         it("should not print the results from previous execution", function() {
-            cli.execute("--no-ignore tests/fixtures/missing-semicolon.js");
+            var configPath = getFixturePath("configurations", "semi-error.json");
+            var filePath = getFixturePath("missing-semicolon.js");
+            var passingPath = getFixturePath("missing-semicolon.js");
+
+            cli.execute("--no-ignore --config " + configPath + " " + filePath);
             assert.isTrue(console.log.called, "Log should have been called.");
 
             console.log.reset();
 
-            cli.execute("--no-ignore tests/fixtures/passing.js");
+            cli.execute("--no-ignore " + passingPath);
```

Comment:
You want to continue to pass in config as this test is passing right now as no rules are on.